src/app/views/welcome/welcome.styl

```diff
@@ -0,0 +1,45 @@
+@require '~styles/variables'
+@require '~styles-lib/mixins'
+
+.-welcome-page
+	theme-prop('background-color', 'darkest')
+	position: absolute
+	top: 0
+	left: 0
+	right: 0
+	bottom: 0
+	overflow-y: auto
+	overflow-x: hidden
+
+.-welcome-col
+	display: flex
+	flex-direction: column
+	align-items: stretch
+	//
+	max-width: 500px
+	margin: 0 auto
+	padding: $grid-gutter-width-xs
```

Comment:
The padding around edges should actually be half the gutter width for phones and desktop

Reply:
Yeah, then isnt this correct? for media-sm-up i apply the grid gutter, otherwise the xs gutter.